frontend/integration-tests/tests/kubevirt/vm.overview.scenario.ts

```diff
@@ -123,4 +122,67 @@ describe('Test vm overview page', () => {
       });
     });
   });
+
+  describe('Test vm overview on project overview page', () => {
+    const newVMDescription = 'edited vm description';
+    beforeEach(async() => {
+      await browser.get(`${appHost}/overview/ns/${testName}`);
+      await isLoaded();
+    });
+
+    it('Check vm details on project overview', async() => {
+      expect(vmView.statusIcon(vmView.statusIcons.off).isPresent()).toBeTruthy();
+      await itemVirtualMachine.click();
+      await isLoaded();
+      await vm.action('Start');
+
+      expect(vmView.vmDetailDesc(testName, vmName).getText()).toEqual(newVMDescription);
+      expect(vmView.vmDetailOS(testName, vmName).getText()).toEqual(basicVmConfig.operatingSystem);
+      expect(vmView.statusIcon(vmView.statusIcons.running).isPresent()).toBeTruthy();
+      expect(vmView.vmDetailTemplate(testName, vmName).getText()).toEqual('openshift/rhel7-generic-small');
+      expect(vmView.vmDetailNamespace(testName, vmName).$('a').getText()).toEqual(testName);
+      expect(vmView.vmDetailBootOrder(testName, vmName).getText()).toEqual(['rootdisk', 'nic0', 'cloudinitdisk']);
+      expect(vmView.vmDetailFlavorDesc(testName, vmName).getText()).toEqual('2 CPU, 4G Memory');
+      expect(vmView.vmDetailIP(testName, vmName).getText()).toContain('10');
+      expect(vmView.vmDetailHostname(testName, vmName).getText()).toEqual(vmName);
+      expect(vmView.vmDetailPod(testName, vmName).$('a').getText()).toContain('virt-launcher');
```

Comment:
Can we construct the whole name?

Reply:
The vm pod has a random string, which is hard to construct. I think it's safe by just verifying it has string `virt-launcher`.
Of course it's better to retrieve vmi pod and call `toEqual` here, but it's not a strong requirement to me.